### PR TITLE
Refine prometheus scrape in FAQ

### LINKED ISSUE #13789: [Documentation] Issue regarding FAQ prometheus scrape
As https://istio.io/latest/about/faq/metrics-and-logs/#prometheus-application-metrics mentioned, the istio-telemetry already removed.
Now we refer https://istio.io/latest/docs/ops/integrations/prometheus/, just scrape data it mentioned. Shall we remove this part of just link it  the prometheus doc?

PS: https://github.com/istio/istio/blob/master/samples/addons/prometheus.yaml#L19

### PR DESCRIPTION
**Pull request by Rei1010**
Please provide a description for what this PR is for.

fix #13789

And to help us figure out who should review this PR, please 
put an X in all the areas that this PR affects.

- [ ] Ambient
- [X] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Extensions and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

### CODE REVIEW

**Comment by Rei1010** [comment]
@kfaseela @zirain @nicole-lihui  ptal

**Review by zirain — APPROVED** [review]
lgtm, thanks!

File: content/en/about/faq/metrics-and-logs/prometheus-application-metrics.md [hunk 1]
```diff
@@ -3,17 +3,11 @@ title: Can I use Prometheus to scrape application metrics with Istio?
...
-Both jobs require that the following annotations are added to any deployments from which application metric collection is desired:
-
-- `prometheus.io/scrape: "true"`
-- `prometheus.io/path: "<metrics path>"`
-- `prometheus.io/port: "<metrics port>"`
+Yes. [Prometheus](https://prometheus.io/) is an open source monitoring system and time series database.
+You can start with [configuration for Prometheus](/docs/ops/integrations/prometheus/#Configuration) that enables collection of application.
+So that you can use Prometheus with Istio to record metrics that track the health of Istio and of
+applications within the service mesh. You can visualize metrics using tools like
+[Grafana](/docs/ops/integrations/grafana/) and [Kiali](/docs/tasks/observability/kiali/).
```
Comment (kfaseela) [inline content/en/about/faq/metrics-and-logs/prometheus-application-metrics.md:10]:
```suggestion
Yes. [Prometheus](https://prometheus.io/) is an open source monitoring system and time series database.
You can use Prometheus with Istio to record metrics that track the health of Istio and of
applications within the service mesh. You can visualize metrics using tools like
[Grafana](/docs/ops/integrations/grafana/) and [Kiali](/docs/tasks/observability/kiali/). See [Configuration for Prometheus](/docs/ops/integrations/prometheus/#Configuration) to understand how to enable collection of metrics.
```

**Review by kfaseela — APPROVED** [review]

**Pull request merged**